Site/SiteAMQPModule.php

```diff
@@ -216,17 +222,17 @@ public function doSyncNs($namespace, $exchange, $message,
 		// callback function must return true or false in order to be handled
 		// correctly by the AMQP extension. If an exception is thrown in this
 		// callback, behavior is undefined.
-		$callback = function(AMQPEnvelope $envelope, AMQPQueue $queue)
-			use (&$response, $correlation_id)
-		{
+		$callback = function (AMQPEnvelope $envelope, AMQPQueue $queue)
+ use (&$response, $correlation_id) {
```

Comment:
Never seen this before, should we change this in the coding standard?  Line with no indentation doesn't look very good